scripts/msct_types.py

```diff
@@ -852,3 +853,11 @@ def compare_centerline(self, other, reference_image=None):
         std = np.std(distances)
         max = np.max(distances)
         return mse, mean, std, max, distances
+
+class flagTypes(Enum):
```

Comment:
we are trying to get away from having importable libraries inside `scripts/`, so i would suggest to move this inside `spinalcordtoolbox/utils.py`

Reply:
Ok sounds good! Changes applied in commit [16a7a8f](https://github.com/neuropoly/spinalcordtoolbox/commit/16a7a8f58cb839af347fb932715cdf31ca1041c0) (addition of Enum class `Metavar` to `spinalcordtoolbox/utils.py`) and removed it from `msct_types` in commit [96d4f3f](https://github.com/neuropoly/spinalcordtoolbox/commit/96d4f3febe741022197f44f5c83c976a4e3d417e).